lib/paper_trail/events/base.rb

```diff
@@ -148,7 +152,7 @@ def changes_in_latest_version
       #
       # @api private
       def ignored_attr_has_changed?
-        ignored = @record.paper_trail_options[:ignore] + @record.paper_trail_options[:skip]
```

Comment:
Here it was only considering ignored attributes defined in the array as symbols and not Hashes.